logstash-core/logstash-core.gemspec

```diff
@@ -47,17 +47,17 @@ Gem::Specification.new do |gem|
 
   gem.platform = "java"
 
-  gem.add_runtime_dependency "pry", "~> 0.10.1"  #(Ruby license)
+  gem.add_runtime_dependency "pry", "~> 0.12.2"  #(Ruby license)
   gem.add_runtime_dependency "stud", "~> 0.0.19" #(Apache 2.0 license)
   gem.add_runtime_dependency "clamp", "~> 0.6.5" #(MIT license) for command line args/flags
-  gem.add_runtime_dependency "filesize", "0.0.4" #(MIT license) for :bytes config validator
-  gem.add_runtime_dependency "gems", "~> 0.8.3"  #(MIT license)
-  gem.add_runtime_dependency "concurrent-ruby", "~> 1.0", ">= 1.0.5"
+  gem.add_runtime_dependency "filesize", "~> 0.2.0" #(MIT license) for :bytes config validator
+  gem.add_runtime_dependency "gems", "~> 1"  #(MIT license)
```

Comment:
my reading of `~> 1` is functionally the same as `>= 1`, meaning if a 2.x is ever released (presumably with breaking changes of some sort), we will blindly consume it. Is this what we intend?